SQF/dayz_code/compile/player_switchModel.sqf

```diff
@@ -9,8 +8,9 @@ disableSerialization;
 _position = player modeltoWorld [0,0,0];
 _dir = getDir player;
 _currentAnim = animationState player;
-//_currentCamera = cameraView;
+_currentCamera = cameraView;
 _playerUID = getPlayerUID player;
+if (Z_SingleCurrency) then {_coins = player getVariable [Z_moneyVariable,0];};
```

Comment:
Actually, I do it here in switch model in my code since you're handling gear here as well. It doesn't really matter that it runs before coins are set in player monitor b/c it gets set anyways later. It works well in switch model.